### DISCUSSION TITLE
Daily 11/10

### DISCUSSION THREAD

**Discussion by rpgrca** [rpgrca/proyecto-ispc-fullstack#114] [General]
Hoy armamos una llamada para empezar con el análisis del sistema cuyas conclusiones estamos plasmando en otra discusión (de ahí habría que llevarlo a la Wiki con explicaciones más profundas y tal vez diagramas y de la Wiki a un pdf en Documentación como hicimos con el ERS). Intentaré hacer otra llamada hoy a la noche o mañana para terminar con la parte de análisis para el miércoles y empezar el diseño también mañana.

**Comment by rocappellano** [comment]
Asistí a llamada, y corregí un par de errores que cometí en el Menú de Navegación, CSS

**Comment by NataliaAlvarezIspc** [comment]
El fin de semana estuve haciendo retoques en  los html y css de los formularios de registración y login, me encontre con algunas trabas, algunas las pude superar, otras tengo que seguir investigando. Además estuve viendo los videos y recomendaciones sobre MVC. Tambien estuve haciendo el favicon para agregarselo a los html. Participe en la reunión virtual del domingo y también en la de hoy martes de análisis del sistema. 
Hoy estaré subiendo el favicon, el logo y los html y css al github y empezaré a trabajar sobre el index.

**Comment by claubott** [comment]
Anoche subí mi rama con el css para la página de contacto anónimo. Hay una discusión abierta sobre los estilos, así que es algo que luego tendré que adaptar. También subí un favicon para practicar, pero era formato .ico y no tendría que haber conflicto con el que subió Natalia. Luego también lo reemplazaré por la versión consensuada. Nota extra: intenté usar un favicon .svg y me funcionó en chrome pero no en firefox! ico o png son mejores opciones.

**Comment by Brian-Almada** [comment]
Ahora estoy escuchando de manera asincrónica la reunión del grupo que se realizó por la mañana (se habló sobre MVC). Luego voy a ver un poquito más de Bootstrap, voy a hacer otra lección de ingles y seguir avanzando con el css de la issue que me asigné.
